server/routes/users.js

```diff
@@ -28,4 +28,13 @@ router.put('/:id', checkRole(Array(roles.at(0))), async (req, res) => {
   }
 });
 
+router.delete('/', async (req, res) => {
```

Comment:
you would receive `id` from `req.query`, to delete specific entity by it's `_id`